hknweb/tours/views.py

```diff
@@ -0,0 +1,54 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.views import generic
+from django.core.mail import EmailMultiAlternatives
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.views.generic.edit import FormView
+from .forms import *
+from django.contrib import messages
+from .models import DepTour
+
+def index(request):
+    tour = DepTour.objects
+
+    context = {
+        'tour': tour,
+    }
+    return render(request, 'tours/index.html', context)
+
+def send_request_email(request, form):
+        subject = 'Department Tour Request'
+        officer_email = 'deprel@hkn.mu'
```

Comment:
This works, but I'd probably go with the `hkn.eecs.berkeley.edu` domain here instead as it's just the more canonical name. Doesn't really matter for the functionality itself though. Same with the text below saying to email if there's a validation error